models/ncgamesmodel.js

```diff
@@ -24,14 +24,34 @@ exports.fetchReviewByID = (reviewID) => {
       });
   }
 };
-exports.fetchReviews = () => {
+exports.fetchReviews = (sort_by, order, category) => {
+  const valuesArr = [];
+
+  //valuesArr.push(order);
+
+  if (!category) {
+    return connection
+      .query(
+        `SELECT reviews.*, count(comments.review_id) as "comment_count" 
+  FROM reviews 
+  LEFT OUTER JOIN comments on reviews.review_id =comments.review_id
+  GROUP BY reviews.review_id
+  ORDER BY ${sort_by} ${order};`,
+        valuesArr
+      )
+      .then((results) => {
+        return results.rows;
+      });
+  }
   return connection
     .query(
       `SELECT reviews.*, count(comments.review_id) as "comment_count" 
   FROM reviews 
   LEFT OUTER JOIN comments on reviews.review_id =comments.review_id
+  WHERE category = ${category}
   GROUP BY reviews.review_id
-  ORDER BY created_at desc;`
+  ORDER BY ${sort_by} ${order};`,
+      valuesArr
     )
```

Comment:
You need to dry this code up using some conditional logic, the complex queries notes are good for this (focusing on optional WHERE clauses) 

You should only be making one db.query in this model but the query you pass in should be conditionally built up depending on whether there is a category present or not. 

Also you can use `$1` parameters in the WHERE clause so I'd do that rather than `${category}` but ORDER BY is fine as it is in your code :D